src/api.internal.d.ts

```diff
@@ -18,16 +18,6 @@ declare module './api' {
          */
         onDidChangeProvider: Event<void>;
     }
-    export interface JupyterServerProvider {
-        /**
-         * Display a `trash` icon next to each server in the quick pick.
-         * Allowing the user to remove this server.
-         * Currently used only by the Jupyter Extension.
-         * A better more generic way to deal with this would be via commands.
-         */
-        removeJupyterServer?(server: JupyterServer): Promise<void>;
```

Comment:
Made this a proposed API, we have 2 usages of this, we can always leave as proposed for now.
Thats cleaner, and easier way to get types (via npm) into the hub extension